client/layout/guided-tours/positioning.js

```diff
@@ -16,20 +16,46 @@ const MASTERBAR_HEIGHT = 47;
 
 const middle = ( a, b ) => Math.abs( b - a ) / 2;
 
-const wouldBeOffscreen = pos =>
-	pos < 0 || ( pos + DIALOG_PADDING + DIALOG_WIDTH ) >
+const wouldBeOffscreen = ( pos ) => {
+	const res = pos < 0 || ( pos + DIALOG_PADDING + DIALOG_WIDTH ) >
 		document.documentElement.clientWidth;
+	return res;
+};
```

Comment:
Are we leaving the explicit `return` intentionally? (that is something I can live with :P, I just wanted to check)